pkg/common/utils.go

```diff
@@ -63,6 +63,9 @@ func GetPartitionNameWithoutClusterID(partitionName string) string {
 }
 
 func WaitFor(interval time.Duration, timeout time.Duration, condition func() bool) error {
```

Comment:
Could we keep one of `WaitForCondition` or `WaitFor`? The k8shim module has `WaitForCondition`, and so maybe we can keep `WaitForCondition` for consistency.

https://github.com/apache/yunikorn-k8shim/blob/58adfe941d2d8dae5544af8b49e435f304678807/pkg/common/utils/utils.go#L245

Reply:
I'm OK with either approach.